api/models/storeDao.js

```diff
@@ -0,0 +1,42 @@
+const appDataSource = require('./dataSource')
+
+const createStoreInfo = async(storeData, res) => {
```

Comment:
```suggestion
const createStoreInfo = async(storeData) => {
```
res는 인자로 받을 필요가 없습니다.